core/api/manifest-parser/src/manifest.ts

```diff
@@ -76,7 +76,7 @@ const engineDecoder = Decode.type({
             try {
                 return Decode.success(new Range(string));
             } catch (err) {
-                return Decode.failure(string, err.message);
+                return Decode.failure(string, (err as Error).message);
```

Comment:
So what's this for? If there's no compile error - the runtime behaviour should be about the same with or without this. Accessing `err.message` when that property doesn't exist will throw a runtime error, but so would a failed cast from `err` to `Error`, no?

Reply:
Nevermind, got it now that I actually read the link :D So _this_ is the actual fix.